requirements.txt

```diff
@@ -1,7 +1,31 @@
-ops
+# This file is managed centrally by release-tools and should not be modified
+# within individual charm repos.  See the 'global' dir contents for available
+# choices of *requirements.txt files for OpenStack Charms:
+#     https://github.com/openstack-charmers/release-tools
+#
+
+# NOTE: newer versions of cryptography require a Rust compiler to build,
+# see
+# * https://github.com/openstack-charmers/zaza/issues/421
+# * https://mail.python.org/pipermail/cryptography-dev/2021-January/001003.html
+#
+cryptography<3.4
 jinja2
-git+https://github.com/openstack-charmers/advanced-sunbeam-openstack#egg=ops_sunbeam
 lightkube
 lightkube-models
-python-keystoneclient
-serialized-data-interface
+ops
+git+https://github.com/openstack-charmers/advanced-sunbeam-openstack#egg=ops_sunbeam
+
+python-keystoneclient  # keystone-k8s
+
+# Used for Traefik
+# Note: Remove when traefik-k8s-operator v1 library is released
+serialized_data_interface
```

Comment:
This actually was changed to the jsonschema library in new iterations of this. Since the relation/charm is only in edge/beta they went ahead and took the liberty to switch from serialized_data_interface to jsonschema. I suspect we should update sooner rather than later, but won't hold this review up for this.

Reply:
Makes sense, thanks for confirming